app/components/agent/Index/quick-start.js

```diff
@@ -7,8 +7,6 @@ import classNames from 'classnames';
 import Emojify from '../../shared/Emojify';
 import Panel from '../../shared/Panel';
 
-import 'highlight.js/styles/atom-one-light.css';
```

Comment:
I prefer having it here so it’s very obvious it’s _only_ used here. We can’t prefix it with anything much as I’d like to, so if it’s at least only loaded when this component is, that’s for the best, IMO!